lib/std/start.zig

```diff
@@ -609,7 +609,7 @@ pub fn call_wWinMain() std.os.windows.INT {
 
     // There's no (documented) way to get the nCmdShow parameter, so we're
     // using this fairly standard default.
-    const nCmdShow = std.os.windows.user32.SW_SHOW;
+    const nCmdShow = 5;
```

Comment:
Certainly a better default than just `SW_SHOW` and okay for passing it to `ShowWindow()`, however `nCmdShow` in `WinMain` should contain the real value instead. You can get it like this:

```zig
const cmd_show = if (peb().ProcessParameters.WindowFlags & STARTF_USESHOWWINDOW != 0) peb().ProcessParameters.ShowWindowFlags else SW_SHOWDEFAULT;
```

Also I would be in favor of defining whatever constants are needed locally/leaving a comment, since just saying "5" or "10" isn't very useful.

---

Unrelated, but I think all the kernel32 usage in that file could be replaced by:
```zig
// GetModuleHandleW(null)
peb().ImageBaseAddress

// GetCommandLineW()
peb().ProcessParameters.CommandLine

// ExitProcess(status)
RtlExitUserProcess(status)
```

Reply:
Nice! All of these changes would make a great PR if you're up for it.

(I was also looking into this and found the PEB solution but missed the `STARTF_USESHOWWINDOW` wrinkle :+1:)